documentation/catalogs/cm_model_catalog.md

```diff
@@ -0,0 +1,36 @@
+| Model Name | Algorithm | Target | Input Features | Non-default Hyperparameters | Forecasting Type | Implementation Status | Implementation Date | Author |
```

Comment:
I'm not sure about "Non-default Hyperparameters". Our task is so specific that it would be a bit remarkable if any model ends up with the default... On the other hand, some hps are very rarely used so this prolly makes sense. Yes, it makes sense